src/main/java/org/matsim/episim/ReplayHandler.java

```diff
@@ -125,9 +132,16 @@ public ReplayHandler(Map<DayOfWeek, List<Event>> events) {
 	 * Replays event add modifies attributes based on current iteration.
 	 */
 	public void replayEvents(final EventsManager manager, DayOfWeek day) {
```

Comment:
If the manager is not needed anymore, maybe the handler can be passed as argument instead?

Reply:
Yep, of course, this is much better solution. I close this pull request and will create a new one.